DeepLinkKit/Router/DPLDeepLinkRouter.m

```diff
@@ -137,6 +137,19 @@ - (BOOL)handleURL:(NSURL *)url withCompletion:(DPLRouteCompletionBlock)completio
 }
 
 
+- (nullable DPLDeepLink *)deepLinkForURL:(nonnull NSURL *)url {
```

Comment:
This method could be used to implement the code above on Line 120 -- code re-use FTW and ensures their implementation doesn't diverge.

ALSO.. helps with coverage